src/apphosting/githubConnections.ts

```diff
@@ -90,13 +90,32 @@ const ADD_ACCOUNT_CHOICE = "@ADD_ACCOUNT";
 const MANAGE_INSTALLATION_CHOICE = "@MANAGE_INSTALLATION";
 
 /**
- * Prompts the user to link their backend to a GitHub repository.
+ * Prompts the user to create a GitHub connection.
  */
-export async function linkGitHubRepository(
+export async function createGitHubConnection(
```

Comment:
Nit: should this be called `getOrCreateGithubConnection` since we'll be returning an existing connection if a connection with the `connectionId` exists?

Reply:
SGTM